src/main.ts

```diff
@@ -35,10 +36,28 @@ const oidcWarning =
   `GitHub Actions workflow permissions are incorrect, or this job is being ` +
   `run from a fork. For more information, please see https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token`;
 
+const headWarning =
```

Comment:
This could be something we can put into utils too so we can just have this message in a single place.

Reply:
https://github.com/google-github-actions/actions-utils/pull/2